lib/controllers/ephemeral/profile_controller.dart

```diff
@@ -1,10 +1,10 @@
 import 'package:get/get.dart';
 import 'package:pispapp/controllers/app/auth_controller.dart';
-import 'package:pispapp/routes/custom_navigator.dart';
+import 'package:pispapp/routes/app_navigator.dart';
 
 class ProfileController extends GetxController {
   Future<void> onLogout() async {
     await Get.find<AuthController>().signOut();
-    Get.find<CustomNavigator>().offAllNamed('/');
+    Get.find<AppNavigator>().offAllNamed('/');
```

Comment:
Actually, what is the purpose of the CustomNavigator/AppNavigator, can we move this? Doesn't even save that much typing

Reply:
Uhmm... I just think that the name `AppNavigator` would make more sense since it is probably intended to handle all of the routing across various parts in the app. The name `CustomNavigator` may confuse people as to whether there's a non-customised version and if there are other navigators in the app.

Reply:
My problem isn't with the naming - its why we are even using this Navigator class at all. It only has one function which is offAllNamed() and when you check the implementation of that - it doesn't do much.

Reply:
Yeah, it was already there before I started the refactoring, but yeah I think it could be a good thing for future iterations to have a wrapper like this. Just need to use enum instead of string for the wrapper.